### PR TITLE
added character and writing ratings

### PR DESCRIPTION
**Pull request by sachiesharma**

### CODE REVIEW

File: src/ReviewEdit.js [hunk 1]
```diff
@@ -32,17 +44,21 @@ export default function ReviewEdit() {
   });
 
   const [content, setContent] = useState("");
+  const [characterRating, setCharacterRating] = useState("");
```
Comment (shadanan) [inline src/ReviewEdit.js:47]: Good variable name!

File: src/ReviewEdit.js [hunk 2]
```diff
@@ -15,6 +18,15 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+const StyledRating = withStyles({
+  iconFilled: {
+    color: "#dda0dd",
```
Comment (shadanan) [inline src/ReviewEdit.js:23]: Good choice of color! Loved the pink!

**Review by shadanan — APPROVED** [review]

**Comment by zianwar** [comment]
Looks good to me as well! 👏👏

File: src/ReviewEdit.js (hunk 1 above)
Comment (sachiesharma) [reply src/ReviewEdit.js:47]: Thanks Dun!! :)

**Pull request merged**